packages/react-native/cli.js

```diff
@@ -67,7 +67,7 @@ async function main() {
       // Ignore errors, since it's a nice to have warning
     }
   }
-  return cli.run();
+  return cli.run(name);
```

Comment:
FYI, this change is intended to reduce fork surface for out of tree platforms like tvos, visionos, macos, etc, that would like to leverage CLI's `init` command without modifications.